spec/index.html

```diff
@@ -539,9 +539,7 @@ <h2>RDF Graphs</h2>
 
   <p>An <dfn>RDF graph</dfn> is a set of <a>RDF triples</a>.</p>
 
-  <p class=issue>
-    Do we still need to define <dfn>asserted triple</dfn>?.
-  </p>
+  <p>An RDF triple is said to be <dfn="asserted">asserted</dfn> in an RDF graph if it is an element of the RDF graph.</p>
```

Comment:
You probably want to use the following:

```suggestion
  <p>An RDF triple is said to be <dfn data-lt="asserted|asserted triple">asserted</dfn> in an RDF graph if it is an element of the RDF graph.</p>
```